sample-local.settings.json

```diff
@@ -1,11 +1,11 @@
 {
     "IsEncrypted":false,
     "Values": {
-        "AzureWebJobsStorage":"",
+        "AzureWebJobsStorage":"UseDevelopmentStorage=true",
         "FUNCTIONS_WORKER_RUNTIME":"python",
-        "STORAGE_CONN_STRING":"",
+        "STORAGE_CONN_STRING":"DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://azurite:10001/devstoreaccount1;BlobEndpoint=http://azurite:10000/devstoreaccount1;",
```

Comment:
no secrets in the repo pls

Reply:
Sorry wasn't clear - this isn't a secret as such, but it is the default connection string for Azurite: https://learn.microsoft.com/en-us/azure/storage/common/storage-use-emulator#authorize-with-shared-key-credentials.
Like I could clear it, and then point to those docs, but I feel a cleaner dev experience is to have it pre-existing - it's no different than having a connection string for a database.

Reply:
Oh right is that key the same no matter what then?

Reply:
Yep it's the default to connect to the emulator, so just an default open secret really. See: https://github.com/search?q=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq&type=code

Reply:
Can't we use `UseDevelopmentStorage=true`?